designs/lily/src/back.mjs

```diff
@@ -0,0 +1,573 @@
+      })
+      macro('vd', {
+        from: points.floorOut,
+        to: points.styleWaistOutLily,
+        x:
+          (points.seatOut.x > points.styleWaistOutLily.x ? points.seatOut.x : points.styleWaistOutLily.x) +
+          sa +
+          15,
+      })
+      macro('vd', {
+        from: points.floorIn,
+        to: points.fork,
+        x: points.fork.x - sa - 15,
+      })
+      macro('vd', {
+        from: points.fork,
+        to: points.styleWaistInLily,
+        x: points.fork.x - sa - 15,
+      })
+      macro('vd', {
+        from: points.floorIn,
+        to: points.styleWaistInLily,
+        x: points.fork.x - sa - 30,
+      })
+      macro('vd', {
+        from: points.crossSeamCurveStart,
+        to: points.styleWaistInLily,
+        x: points.crossSeamCurveStart.x - sa - 15,
+      })
+      macro('hd', {
+        from: points.styleWaistInLily,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 15,
+      })
+      macro('hd', {
+        from: points.crossSeamCurveStart,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 30,
+      })
+      macro('hd', {
+        from: points.crossSeamCurveMax,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 45,
+      })
+      macro('hd', {
+        from: points.fork,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 60,
+      })
+      macro('hd', {
+        from: points.grainlineTop,
+        to: points.styleWaistOutLily,
+        y: points.styleWaistInLily.y - sa - 15,
+      })
+      if (points.seatOut.x > points.styleWaistOutLily.x) {
+        macro('hd', {
+          from: points.grainlineTop,
+          to: points.seatOut,
+          y: points.styleWaistInLily.y - sa - 30,
+        })
+      }
+    }
+  }
+
+  return part
+}
+
+export const back = {
+  name: 'lily.back',
+  from: titanBack,
+  //after: titanFront,
+  measurements: [
+    'ankle',
+    'heel', // secondary measurement, used instead of ankle
+  ],
+  options: {
+    fitGuides: {bool: true, menu: undefined},
+    fitKnee: {bool: true, menu: undefined},
+    legBalance: 0.5, // between back and front parts
+    waistBalance: 0.5,
+    crotchDrop: { pct: 0, min: 0, max: 15, menu: 'advanced' }, // 'downgrade' to advanced menu
+    waistHeight: { ...titanBack.options.waistHeight, pct: 50 }, // halfway between waist and hips
+    fabricStretch: { pct: 40, min: 0, max: 50, menu: 'fit' },
+    waistEase: {pct: -4, min: 0, max: 50, menu: 'fit' }, // -fabricStretch/10,
+    seatEase: {pct: -4, min: 0, max: 50, menu: 'fit' }, // -fabricStretch/10,
+    kneeEase: {pct: -4, min: 0, max: 50, menu: 'fit' }, // -fabricStretch/10,
+    //test: {pct: back.options.fabricStretch/2, min: 0, max: 50, menu: 'fit'},
+    lengthBonus: 0,
+    lengthReduction: { pct: 0, min: 0, max: 100, menu: 'style'},
+    waistbandWidth: { ...titanBack.options.waistbandWidth, menu: 'style' },
+    },
+  hideDependencies: true,
```

Comment:
The parts config API has since changed for hiding parts and dependencies. Please see https://freesewing.dev/reference/api/part/config/hide .
I think the updated config would be: `hide: 'HIDE_TREE',`

You'll probably want to change it here and in other files.